homeassistant/components/victron_remote_monitoring/strings.json

```
@@ -0,0 +1,92 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Enter your VRM API token. We will then fetch your available sites.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "The API key for your VRM account"
```

You might want to adapt this here as well and use "API token" instead.

```suggestion
        "data": {
          "api_key": "[%key:common::config_flow::data::api_token%]"
        },
        "data_description": {
          "api_key": "The API access token for your VRM account"
```

In that case changing the key from "api_key" to "api_token" here and in code should be best.